gutenberg-starter-theme-blocks/experimental-theme.json

```diff
@@ -0,0 +1,204 @@
+				{
+					"slug": "small",
+					"value": "13px"
+				},
+				{
+					"slug": "normal",
+					"value": "16px"
+				},
+				{
+					"slug": "medium",
+					"value": "20px"
+				},
+				{
+					"slug": "large",
+					"value": "36px"
+				},
+				{
+					"slug": "huge",
+					"value": "48px"
+				}
+			],
+			"line-height": [
+				{
+					"slug": "small",
+					"value": "1.2"
+				},
+				{
+					"slug": "medium",
+					"value": "1.4"
+				},
+				{
+					"slug": "large",
+					"value": "1.8"
+				}
+			]
+		},
+		"styles": {
+			"color": {
+				"background": "var( --wp--preset--color--background )",
+				"text": "var( --wp--preset--color--foreground )"
+			}
+		}
+	},
+	"core/paragraph": {
+		"styles": {
+			"color": {
+				"background": "var( --wp--preset--color--background )",
+				"link": "var( --wp--preset--color--primary )",
+				"text": "var( --wp--preset--color--foreground )"
+			},
+			"typography": {
+				"fontSize": "var( --wp--preset--font-size--normal )",
+				"lineHeight": "var( --wp--preset--line-height--large )"
+			}
+		}
+	},
+	"core/heading/h1": {
+		"styles": {
+			"color": {
+				"background": "var( --wp--preset--color--background )",
+				"link": "var( --wp--preset--color--primary )",
+				"text": "var( --wp--preset--color--foreground )"
+			},
+			"typography": {
+				"fontSize": "var( --wp--preset--font-size--huge )",
+				"lineHeight": "var( --wp--preset--line-height--small )"
+			}
+		}
+	},
+	"core/heading/h2": {
+		"styles": {
+			"color": {
+				"background": "var( --wp--preset--color--background )",
+				"link": "var( --wp--preset--color--primary )",
+				"text": "var( --wp--preset--color--foreground )"
+			},
+			"typography": {
+				"fontSize": "var( --wp--preset--font-size--large )",
+				"lineHeight": "var( --wp--preset--line-height--small )"
+			}
+		}
+	},
+	"core/heading/h3": {
+		"styles": {
+			"color": {
+				"background": "var( --wp--preset--color--background )",
+				"link": "var( --wp--preset--color--primary )",
+				"text": "var( --wp--preset--color--foreground )"
+			},
+			"typography": {
+				"fontSize": "var( --wp--preset--font-size--medium )",
+				"lineHeight": "var( --wp--preset--line-height--medium )"
+			}
+		}
+	},
+	"core/heading/h4": {
+		"styles": {
+			"color": {
+				"background": "var( --wp--preset--color--background )",
+				"link": "var( --wp--preset--color--primary )",
+				"text": "var( --wp--preset--color--foreground )"
+			},
+			"typography": {
+				"fontSize": "var( --wp--preset--font-size--normal )",
+				"lineHeight": "var( --wp--preset--line-height--medium )"
+			}
+		}
+	},
+	"core/heading/h5": {
+		"styles": {
+			"color": {
+				"background": "var( --wp--preset--color--background )",
+				"link": "var( --wp--preset--color--primary )",
+				"text": "var( --wp--preset--color--foreground )"
+			},
+			"typography": {
+				"fontSize": "var( --wp--preset--font-size--small )",
+				"lineHeight": "var( --wp--preset--line-height--large )"
+			}
+		}
+	},
+	"core/heading/h6": {
```

Comment:
Just noting the redundancy of some of these styles. It becomes really verbose, and I don't think we want to end up with a json config that's just as long as a theme stylesheet, without a way to compile / generate it.

It would be nice to have a way target multiple blocks with the same styles, like `"core/heading/h*": {`